recipes-ni/sysconfig-settings-ssh/sysconfig-settings-ssh_1.0.bb

```diff
@@ -0,0 +1,28 @@
+SUMMARY = "System configuration files for ssh"
+DESCRIPTION = "SSH configuration files for the National Instruments System Configuration subsystem."
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
+SECTION = "base"
+
+SRC_URI = "file://uixml/nilinuxrt.sshd_enable.binding.xml \
+           file://uixml/nilinuxrt.sshd_enable.const.de.xml \
+           file://uixml/nilinuxrt.sshd_enable.const.fr.xml \
+           file://uixml/nilinuxrt.sshd_enable.const.ja.xml \
+           file://uixml/nilinuxrt.sshd_enable.const.ko.xml \
+           file://uixml/nilinuxrt.sshd_enable.const.xml \
+           file://uixml/nilinuxrt.sshd_enable.const.zh-CN.xml \
+           file://uixml/nilinuxrt.sshd_enable.def.xml \
+"
+
+FILES_${PN} += "/usr/local/natinst/share/uixml/sysconfig/* \
+"
+
+DEPENDS += "shadow-native pseudo-native niacctbase"
```

Comment:
None of these files are owned by `lvuser`. Why does this recipe need to depend upon `niacctbase`?

Reply:
Carryover from another recipe I based this on. Its not required. Removed.